src/main.rs

```diff
@@ -43,6 +79,37 @@ fn main() {
                 Err(e) => println!("{}", e.to_string()),
             }
         }
+        ("import", args) => {
```

Comment:
If you make this `("import", Some(args))` you can avoid the "unwrap_or_else" in next statement.

Reply:
Also pass the entire `args` to `ft_cli::Config::from_args` maybe?